operator/v1/types_ingress.go

```diff
@@ -1054,6 +1054,22 @@ type ContainerLoggingDestinationParameters struct {
 }
 
 // LoggingDestination describes a destination for log messages.
+// + ---
```

Comment:
Is the `// + ` a way to include a comment in the file but not have it show up in `oc explain` ?  Nifty.

Reply:
Indeed.  Every line with the `// +` prefix is omitted from the CRD/`oc explain` output, and everything after `// + ---` is omitted as well.  (I suppose using `+` after the `// + ---` line is redundant, but I see it used in other cases where we have a preceding `// + ---`.)  https://github.com/openshift/enhancements/blob/master/dev-guide/api-conventions.md#write-user-readable-documentation-in-godoc has examples and details.